src/LiveQuery/RequestSchema.js

```diff
@@ -70,6 +70,22 @@ const subscribe = {
           minItems: 1,
           uniqueItems: true,
         },
+        keys: {
```

Comment:
A thought... depending on how the Server uses this schema (I don't know if the schema is removed when an unsubscribe happens or when a server is shut down, I wonder if you can remove `fields` from the schema entirely without breaking older servers that upgrade? If the schema is maintained after shutdown, you can simply migrate it to a schema without fields when upgrading.

Reply:
I'm thinking that as long as the key `fields` hasn't been deprecated, then it should stay in the schema. Updating all the SDKs to use `keys` might take some time

Reply:
I figured you already changed and deprecated the need for fields in the schema here: https://github.com/parse-community/parse-server/blob/c21257f8264598ecfd1c02e09e642cdd9fcfc02f/src/LiveQuery/ParseLiveQueryServer.js#L905-L911

I assume this line of code means you take `fields` and use it as `keys`? If so, this seems like Client SDK's still using `fields` will work because they never interact with the Schema directly. It would also mean that `fields` in the schema isn't being used on the server anymore because both `keys` and `fields` are stored in keys. I haven't verified this, but skimming the code looks this way. The thing to check what be my comments here about how the schema could be used https://github.com/parse-community/parse-server/pull/8388#discussion_r1083227975

Reply:
> I assume this line of code means you take fields and use it as keys?

Yes that is correct, this is for backwards compatibility to give the SDKs some time to update.

The "RequestSchema" is used to validate the data on _any_ incoming livequery request, and is called before every request in `parseWebsocket.on('message', request => {` so both options will need to be there, to allow incoming requests to have either `watch`, `keys`, or `fields`